code/modules/mob/living/basic/vermin/mouse.dm

```diff
@@ -116,7 +116,7 @@
 	if(!gibbed)
 		var/obj/item/food/deadmouse/mouse = new(loc)
 		mouse.name = name
-		mouse.icon_state = icon_dead
+		mouse.icon_state = "mouse_[body_color]_dead"
```

Comment:
This looks like it's going to make the `splat()` proc not work, since it also sets `icon_dead`.

You should instead set `icon_dead` on Initialize after body_color is set.

Reply:
So it looks like this is already set by /datum/element/animal_variety. The issue we're experiencing here though is this element is only added when body_color is null, would I be correct in thinking the best way to implement this would be to move this element outside of that if check?

Reply:
Yes